boards/arc/emsdp/arc_mpu_regions.c

```diff
@@ -45,6 +45,10 @@ static struct arc_mpu_region mpu_regions[] = {
 			 0xF0000000,
 			 64 * 1024,
 			 REGION_KERNEL_RAM_ATTR),
+	MPU_REGION_ENTRY("FLASH_0",
```

Comment:
Is that flash device already usable? If not maybe remove it?

Reply:
renamed to SPI_1